Server/Components/UI/ItemCard.razor

```diff
@@ -1,12 +1,12 @@
 @using Fracture.Server.Modules.Items.Models
 
 <div class="col">
-    <div class="card" style="width: 15rem;">
-        <div class="card-header" style="background-color: var(@_color)">
+    <div class="card" style="min-width: 13rem;">
+        <div class="card-header" style="background-color: var(--rarity-@Item.Rarity.ToString().ToLower())">
```

Comment:
honestly i would've more liked an extension method on the enum called something like `ToCssClassName()`, instead of doing .ToString().ToLower()

looks and works fine though, just my own nitpicks :P

Reply:
That's a very good point, it doesn't feel clean at all. I'll rewrite it so it's an extension method.